pkgs/top-level/python-packages.nix

```diff
@@ -109,6 +107,7 @@ let
         name = namePrefix + oldAttrs.name;
```

Comment:
Is this correct? E.g. `python.pkgs.attrs.passthru.name` is `python2.7-python2.7-attrs-17.2.0`.

Reply:
Right. That's because in this implementation `toPythonModule` is used directly inside `buildPythonPackage`. Let me remove the `name = namePrefix + oldAttrs.name;` line because its pretty useless anyway.